[doc] add docker instructions to the readme (#558)
DataDog/datadog-ci: README.md

datadog-ci version
```

### Container image (**experimental**)

> In order to run `datadog-ci` from a container, the following `Dockerfile` can be used:

Reviewer: ```suggestion
In order to run `datadog-ci` from a container, you can use the following `Dockerfile:
```
